client/src/pages/Dashboard.jsx

```diff
@@ -67,6 +83,16 @@ export default function Dashboard() {
       .then((data) => setVideos(data));
   }, []);
 
+  useEffect(() => {
```

Comment:
this useEffect already exist above, you can modify it and add the search lesson in the dependency array i think

Reply:
ok fait